cluster/images/hyperkube/Makefile

```diff
@@ -21,7 +21,7 @@ REGISTRY?=gcr.io/google_containers
 ARCH?=amd64
 OUT_DIR?=_output
 TEMP_DIR:=$(shell mktemp -d -t hyperkubeXXXXXX)
-CNI_RELEASE=0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff
+CNI_RELEASE=v0.6.0
```

Comment:
Why this change? CNI and Kubernetes versions have a matrix of compatibility, especially for old versions like 1.7.x.

Or is this a known good revision to use with 1.7.x?

Reply:
this is a known version we have been using with 1.7